### PR TITLE
feat: Update toolchain and tidy up

### PR DESCRIPTION
**Pull request by dailywraith**
Updates dependencies and tidies tools

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,123 +10,119 @@ A Jest Circus environment for Allure reporting.
...
-    - [📈 DocBlocks](#-docblocks)
-        - [🔍 Descriptions](#-descriptions)
-        - [🏷 Tag](#-tag)
-        - [👥 Owner](#-owner)
-        - [:part_alternation_mark: Severity](#part_alternation_mark-severity)
-        - [📇 Behaviors (epics, features, stories)](#-behaviors-epics-features-stories)
-        - [🔗 Links (Jira and TMS)](#-links-jira-and-tms)
-    - [👩‍🎓 Advanced](#-advanced)
-        - [🎛 Global Allure API](#-global-allure-api)
+  - [❗️ Requirements](#-requirements)
```
Comment (dacudo) [inline README.md:16]: This link is broken because of the icon. I think the link needs to be:
`#%EF%B8%8F-requirements`

File: README.md
Comment (dacudo) [inline README.md:1]: L144 delimiter spelling

File: __tests__/allure-api/attachment.test.js [hunk 2]
```diff
@@ -1,26 +1,35 @@
...
+  allure.attachment('PNG-attachment', '', ContentType.PNG);
+  allure.attachment('SVG attachment', '', ContentType.SVG);
+  allure.attachment(
+    'HTML attachment',
+    '<div><p>This is an HTML doc</p></div>',
+    ContentType.HTML,
+  );
 
-	expect(1 + 2).toBe(3);
+  expect(1 + 2).toBe(3);
```
Comment (dacudo) [inline __tests__/allure-api/attachment.test.js:30]: what is this testing?

File: __tests__/allure-api/description.test.js [hunk 3]
```diff
@@ -1,5 +1,4 @@
-
 test('allure.description()', () => {
-	allure.description('This is an example description');
-	expect(5).toBe(5);
+  allure.description('This is an example description');
+  expect(5).toBe(5);
```
Comment (dacudo) [inline __tests__/allure-api/description.test.js:3]: what is this testing?

File: __tests__/allure-api/attachment.test.js (hunk 2 above)
Comment (dailywraith) [reply __tests__/allure-api/attachment.test.js:30]: the tests are designed to test the allure reporting and are not "tests" in the traditional sense. They are used to produce all the different states a test can get in so when you view the report there's at least one test in each

File: README.md (hunk 1 above)
Comment (dailywraith) [reply README.md:16]: interestingly I don't see that error, it works grand in intelliJ

Comment (dacudo) [reply README.md]: If you follow this link or click Requirements in the table of contents, does it bring you to the Requirements section?
https://github.com/dailywraith/jest-circus-allure-environment/blob/66e4f34b959ae748fb302f527ab8742373f9fac7/README.md#-requirements

File: README.md
Comment (dacudo) [reply README.md:1]: fixed

File: __tests__/allure-api/attachment.test.js (hunk 2 above)
Comment (dacudo) [reply __tests__/allure-api/attachment.test.js:30]: This review comment can be closed. There is no Resolve Conversation button to allow me to close it.

File: __tests__/allure-api/description.test.js (hunk 3 above)
Comment (dacudo) [reply __tests__/allure-api/description.test.js:3]: This review comment can be closed. There is no Resolve Conversation button to allow me to close it.

**Review by paulforsythe1 — APPROVED** [review]

**Pull request merged**